st2api/st2api/controllers/v1/aliasexecution.py

```diff
@@ -88,7 +91,19 @@ def post(self, payload):
                                              notify=notify,
                                              context=context)
 
-        return str(execution.id)
+        result = {
```

Comment:
Since we change the status code and hubot-stackstorm catches `200` in the old version, I think it'll yield a "error: status 201" error or something like that. It's not very informative, but at least it's a well-defined error with the only probable cause being the need to update, because nothing else will ever give 201 status at this endpoint. :) As soon as we get a KB, I'll put a message there, of course.

I tried to find a good way to make this backwards-compatible without introducing extra flags or parameters, and, honestly, couldn't, because old hubot expects a string.